### ISSUE TITLE
Cant auth

### ISSUE THREAD

**Issue by la-scent** [bendews/apw#1]
Download apw v0.0.4 from github (try both amd64/arm64 versions) but getting error for every commad which i try.

```sh
$ apw start
error: Uncaught (in promise) Error: No existing keys. Please login first.
    throw new Error("No existing keys. Please login first.");
          ^
    at readConfig (file:///home/runner/work/apw/apw/src/utils.ts:96:11)
    at new ApplePasswordManager (file:///home/runner/work/apw/apw/src/client.ts:149:43)
    at file:///home/runner/work/apw/apw/src/cli.ts:30:16
```

```sh
apw --help
error: Uncaught (in promise) Error: No existing keys. Please login first.
    throw new Error("No existing keys. Please login first.");
          ^
    at readConfig (file:///home/runner/work/apw/apw/src/utils.ts:96:11)
    at new ApplePasswordManager (file:///home/runner/work/apw/apw/src/client.ts:149:43)
    at file:///home/runner/work/apw/apw/src/cli.ts:30:16
```

```sh
$ apw auth
error: Uncaught (in promise) Error: No existing keys. Please login first.
    throw new Error("No existing keys. Please login first.");
          ^
    at readConfig (file:///home/runner/work/apw/apw/src/utils.ts:96:11)
    at new ApplePasswordManager (file:///home/runner/work/apw/apw/src/client.ts:149:43)
    at file:///home/runner/work/apw/apw/src/cli.ts:30:16
```

hardware: 
Chip: Apple M3 Pro
macOS: 14.5 (23F79)

**Comment by bendews** [comment]
So sorry you ran in to this issue! looks like the order of operations are wrong for reading/initiating the configuration on startup. To resolve this manually you can run the following command to create an empty configuration:

`mkdir ~/.apw && echo "{}" > ~/.apw/config.json`

I'll add a proper fix for this issue when i get some free time & leave this issue open until that fix has been pushed.

Thanks for finding & contributing to the project! :)

**Comment by al-scent** [comment]
Yeah that help thanks!

**Comment by bendews** [comment]
Fixed #4 👍
